src/main/java/uk/gov/hmcts/reform/professionalapi/util/RefDataUtil.java

```diff
@@ -432,4 +437,31 @@ public static void checkOrganisationAndPbaExists(Organisation organisation) {
             throw new ResourceNotFoundException(ERROR_MSG_NO_PBA_FOUND);
         }
     }
-}
+
+    public static void checkOrganisationAndMoreThanRequiredAddressExists(

```

Comment:
Any reason for this method  to be in util class? are we calling this method in multiple places?

Reply:
we keep all validation method in util class so that if not today but in future we can re use similar method.